package tz.co.yesayasoftware.authdemo.data.network

class ApiErrorException: Exception()